cmd_vel_to_wheels:
  ros__parameters:
    track_width_m: 0.62
    max_wheel_speed_mps: 1.0
    deadzone: 0.05
    publish_rate_hz: 20.0
    cmd_timeout_ms: 250

teensy_transport_stub:
  ros__parameters:
    transport_timeout_ms: 250
    log_rate_hz: 10.0
